x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx

```diff
@@ -67,36 +67,40 @@ export const getFieldItems = ({
   );
 };
 
-const getDefaultFieldTableColumns = (highlight: string): FieldTableColumns => [
-  {
+const getDefaultFieldTableColumns = ({
+  highlight,
+  shouldShowDescriptionColumn,
+}: {
+  highlight: string;
+  shouldShowDescriptionColumn: boolean;
+}): FieldTableColumns => {
+  const nameColumn = {
     field: 'name',
     name: i18n.NAME,
-    render: (name: string, { type }) => {
-      return (
-        <EuiFlexGroup alignItems="center" gutterSize="none">
-          <EuiFlexItem grow={false}>
-            <EuiToolTip content={type}>
-              <EuiIcon
-                data-test-subj={`field-${name}-icon`}
-                css={styles.icon}
-                type={getIconFromType(type ?? null)}
-              />
-            </EuiToolTip>
-          </EuiFlexItem>
+    render: (name: string, { type }: { type?: string }) => (
```

Comment:
nitpick: is there 1 common type that we can use for the second parameter of the `render` method for all of the columns?